File: 240306/더 긴 문자열/longer-string.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    string str1, str2;
    cin >> str1 >> str2;
    int len1 = str1.length();
    int len2 = str2.length();
    if (len1 == len2){
        cout << "same";
    }
    else if (len1 > len2){
        cout << str1 << " " << len1;
    }
    else{
        cout << str2 << " " << len2;
    }
    return 0;
}

File: 240611/더 긴 문자열/longer-string.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String str1 = sc.next();
        String str2 = sc.next();

        int lenStr1 = str1.length();
        int lenStr2 = str2.length();

        if (lenStr1 == lenStr2) System.out.print("same");
        else if (lenStr1 > lenStr2) System.out.printf("%s %d", str1, lenStr1);
        else System.out.printf("%s %d", str2, lenStr2);

    }
}